cmd/wire/main.go

```diff
@@ -73,7 +73,7 @@ func main() {
 	// Configure Cayley.
 	wire.Println("Configuring Cayley")
 
-	graphDB, err = cayleyshelf.New(mongoURI.String())
+	graphDB, err = cayleyshelf.New(mongoURI.String(), nil)
```

Comment:
You'll need to use the same "session name" as the mongo session that was registered, in this case `mongoURI.Path`.